contents/handbook/growth/sales/expansion-and-retention.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Expansion & Retention
+sidebar: Handbook
+showTitle: true
+---
+
+> The most fundamental thing you should do first is to get multiple contacts at the same place in case our champion leaves. This is the best way to derisk a customer leaving in the future.
+
+## Expansion
+
+AEs also do expansion at PostHog. This is because we are constantly on a sales footing with customers - for the most part, we don't do steady state account management with an arbitrary 10% uplift at renewal team. There are two primary ways we look to drive expansion with a customer:
+
+**Cross sell new products**
+- First you need to find out who cares about the new product - is it the existing team or the new team?
+- Then you need to find out what are they using now for other tools - surface this during the check in calls that you already have scheduled as part of onboarding.
+- At this point
+  - If it's already a mature product we have shipped, you should aim to show how the product complements what they _already_ are using in PostHog - don't just arbitrarily sell in a product for the sake of it. 
+  - If it's something in beta or coming soon, you should start giving them sneak peeks of what's on our [roadmap](/roadmap). You can also schedule a feedback session with the relevant product engineer if they’re a great fit - customers _love_ this. Consider playing the founder card for something _really_ new and big. 
+
+**Multiple teams using the same product**
+
+This is a bit more straightforward, but harder to execute:
+- Make sure you are asking for intros to other teams during the regularly scheduled checkin calls - ‘who else would benefit from this?’, 'are there other teams with similar pain points?'
+- This is harder to do without in-person visits
+
+
+### Principles for visiting customers
+
+If you offer to do a meeting in person with a customer, they’ll then feel obliged to introduce you to other people to make good use of your time. Trying to get them to adopt more products can be a good trigger, but generally you should be matching the cadence for in-person meetings with the size of contract (ie. more regular for Very Large, less regular for Large). 
+
+Generally speaking you should be trying to regularly see customers in your book of business who are $60k+ annually, or could get there. Occasionally you can pull in James/Tim if they are traveling to SF/NY especially. 
```

Comment:
I'm happy to do quite a lot of this next couple of months as a test.

Reply:
@simfish85 @camerondeleone maybe you could share a list here of those you'd think it's worth Tim talking to over the next couple of months? 

In particular I'd be keen to understand whether it's at the new sale/renewal/cross-sell/all stage where in-person is most effective, so a mix of customers could be good. 

(I know @simfish85 you mentioned that most customers don't respond to the CS team and only want to talk to founders - maybe there's a way we can test that, not sure how...)

Reply:
Will do - looking into this this week

Reply:
I'm to resolve this conversation btw @simfish85 just to tidy up the PR, but you can follow up in Slack?